tests/utils/test_io.py

```diff
@@ -33,3 +36,36 @@ def test_valid_filenames(self):
                 else:
                     with self.assertRaises(ValueError):
                         check_filenames(filenames)
+
+class TestEmbeddingsIO(unittest.TestCase):
+
+    def setUp(self):
+        # correct embedding file as created through lightly
+        self.embeddings_path = tempfile.mktemp('.csv', 'embeddings')
+        embeddings = np.random.rand(32, 2)
+        labels = [0 for i in range(len(embeddings))]
+        filenames = [f'img_{i}.jpg' for i in range(len(embeddings))]
+        save_embeddings(self.embeddings_path, embeddings, labels, filenames)
+
+    def test_valid_embeddings(self):
+        check_embeddings(self.embeddings_path)
+
+    def test_whitespace_in_embeddings(self):
+        # should fail because there whitespaces in the header columns
+        lines = ['filenames, embedding_0,embedding_1,labels\n',
+                 'img_1.jpg, 0.351,0.1231']
```

Comment:
I am not sure why you add the 2nd line here, as only the first line is checked. The same in the other test.